package.json

```diff
@@ -43,6 +43,7 @@
     "@babel/plugin-transform-react-constant-elements": "7.10.1",
     "@babel/plugin-transform-react-jsx": "7.10.1",
     "@babel/preset-env": "7.11.0",
+    "@chiragrupani/karma-chromium-edge-launcher": "2.0.0",
```

Comment:
Not sure what the bar even is but is this a reputable package? https://github.com/ChiragRupani/karma-chromiumedge-launcher

Reply:
Yes, it is. The official [`karma-edge-launcher`](https://www.npmjs.com/package/karma-edge-launcher) package hasn't been updated in 3 years, and [does not work](https://github.com/karma-runner/karma-edge-launcher/issues/11) on recent versions of Windows with the chromium-based version of Edge. [This issue](https://github.com/karma-runner/karma-edge-launcher/issues/11) on the official repo links to [`karma-chromium-edge-launcher`](https://www.npmjs.com/package/@chiragrupani/karma-chromium-edge-launcher) which is being actively maintained, and there is an [ongoing discussion](https://github.com/karma-runner/karma-edge-launcher/issues/11#issuecomment-613338318) about merging it into the official repo. Until then, we're going to have to use the new package.